static/app/views/replays/detail/network/index.tsx

```diff
@@ -122,23 +125,17 @@ const Item = styled('div')<{color?: string; numeric?: boolean}>`
   padding: ${space(0.75)} ${space(1.5)};
   background-color: ${p => p.theme.background};
   min-width: 0;
+  line-height: 16px;
 
   ${p => p.numeric && 'font-variant-numeric: tabular-nums;'}
 `;
 
 const StyledPanelTable = styled(PanelTable)<{columns: number}>`
   grid-template-columns: max-content minmax(200px, 1fr) repeat(3, max-content);
   font-size: ${p => p.theme.fontSizeSmall};
-  line-height: 16px;
   margin-bottom: 0;
-  height: 100%;
+  max-height: 100%;
```

Comment:
I'm not feeling this change.

We've got these big panel areas for the content to inflate and fill up.

cc @Jesse-Box 

the screen shot (copied from the description) shows the behavior in the bottom-right corner:

![181275810-3ae25409-d8bb-4f58-af2a-0e37157cd279](https://user-images.githubusercontent.com/187460/181379073-c0da9f7f-cbc9-4283-9559-be50a2b93243.png)

Reply:
The result would be something like this:

![image](https://user-images.githubusercontent.com/39612839/181384910-356e70c7-9ded-4dc5-83e8-1f4227aabe9f.png)

Reply:
Considering this is a scrollable window, I think the 100% height is the way to go because it explains the bounds of the container. @jesus4497 @ryan953

Reply:
```suggestion
  height: 100%;
```

Reply:
I like the borders, I'll make low-pri follow up tickets to make sure all the other sections work this way (and their empty states too)

Reply:
Done! 

![image](https://user-images.githubusercontent.com/39612839/181608932-433e6b21-8494-45a2-8288-6cf97c9cd847.png)

![image](https://user-images.githubusercontent.com/39612839/181609000-d8b53d87-db26-41ed-a56a-7bb75f561432.png)

Reply:
awesome. Thanks @jesus4497 for trying out the `max-height` option and taking a screen shot. I hadn't actually done that so it's really good to try it, make the proposal, and know for sure what the options looks like. 🥇